streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -480,6 +484,13 @@ public void shouldAcceptExactlyOnce() {
         new StreamsConfig(props);
     }
 
+    @Test
+    public void shouldAcceptExactlyOnceBeta() {
+        // don't use `StreamsConfig.EXACLTY_ONCE_BETA` to actually do a useful test
```

Comment:
Could you elaborate what you mean here?

Reply:
If we would use `EXACLTY_ONCE_BETA` in this test, we would compare `EXACLTY_ONCE_BETA == EXACLTY_ONCE_BETA` what is not useful -- we should compare `EXACLTY_ONCE_BETA == "exactly_once_beta"` instead.

Reply:
Makes sense!